### ISSUE TITLE
Add testnet instructions

### ISSUE THREAD

**Issue by mflaxman** [btcguide/btcguide.github.io#21]
Cobo now supports this

**Comment by wrengulf** [comment]
It would be great if we could use testnet on Specter... I find electrum testnet a great way to play around and get comfortable with the setup. Is there a way to do this without setting up a testnet node for specter to connect to ? Some have suggested regnet but I was unsure what that entails.

**Comment by Gilbog** [comment]
I agree, particularly for relative newbies a section on how to learn and practice the setup on testnet would be very helpful. 
Being a newbie, it took me many days to learn and figure out how to do this on testnet, and I'm still not understanding some details.  
The testnet node was the smallest obstacle, takes just 3-4 hours to sync on my old laptop, and the testnet3 folder takes up just 35 GB

**Comment by merland** [comment]
I humbly disagree... I think Testnet is essential for developers to try out new features on, but for newbie users it's an extra complication. One more dimension to get your head around while trying to learn many other difficult concepts. Test on Mainnet with small amounts instead, especially now when fees are very low.  :)

**Comment by Gilbog** [comment]
I decided to try testnet because in his podcast with Stephan Livera, SLP 215, Michael Flaxman recommended to try riskless multisig on testnet.
I see your point, though: After working through testnet functionality, I now understand that testnet comes with its own complications, is not well documented, and thus not newbie-friendly. I learned a lot in the process, and it feels good to acquire multisig experience without fear of messing up -- up to the point where I got stuck (see my issue #100)
I'll follow your advice if I can't figure out the emergency recovery on testnet soon. Thanks for your perspective!
